sdk/recoveryservices/arm-recoveryservices/README.md

```diff
@@ -108,10 +108,6 @@ If you'd like to contribute to this library, please read the [contributing guide
 
 - [Microsoft Azure SDK for JavaScript](https://github.com/Azure/azure-sdk-for-js)
 
-
-
-[azure_cli]: https://learn.microsoft.com/cli/azure
```

Comment:
do you know which tool removed this?

Reply:
I saw duplicate content manually removed